new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -561,18 +571,91 @@ public void testDomainInPvUsingWdt() {
+    assertDoesNotThrow(() -> copyFileToPod(wdtDomainNamespace, serverName, "weblogic-server",
+        killServerScript.toPath(), Paths.get(destLocation)),
+        String.format("Failed to copy file %s to pod %s in namespace %s",
+            killServerScript, serverName, wdtDomainNamespace));
+    logger.info("File copied to Pod {0} in namespace {1}", serverName, wdtDomainNamespace);
+
+    // get the restart count of the container in pod before liveness probe restarts
+    final int beforeRestartCount =
+        assertDoesNotThrow(() -> getContainerRestartCount(wdtDomainNamespace, null, serverName, null),
+            String.format("Failed to get the restart count of the container from pod {0} in namespace {1}",
+                serverName, wdtDomainNamespace));
+    logger.info("Restart count before liveness probe {0}", beforeRestartCount);
+    // change file permissions
+    ExecResult execResult = assertDoesNotThrow(() -> execCommand(wdtDomainNamespace, serverName, null,
+        true, "/bin/sh", "-c", "chmod +x " + destLocation),
+        String.format("Failed to change permissions for file %s in pod %s", destLocation, serverName));
+    assertTrue(execResult.exitValue() == 0,
+        String.format("Failed to change file %s permissions, stderr %s stdout %s", destLocation,
+            execResult.stderr(), execResult.stdout()));
+    logger.info("File permissions changed inside pod");
+    /* First, kill the mgd server process in the container three times to cause the node manager to
+     * mark the server 'failed not restartable'. This in turn is detected by the liveness probe, which
+     * initiates a container restart.
+     */
+    for (int i = 0; i < 3; i++) {
+      execResult = assertDoesNotThrow(() -> execCommand(wdtDomainNamespace, serverName, null,
+          true, "/bin/sh", "-c", destLocation + " " + serverName),
+          String.format("Failed to execute script %s in pod %s namespace %s", destLocation,
+              serverName, wdtDomainNamespace));
+      logger.info("Command executed to kill server inside pod, exit value {0}, stdout {1}, stderr {2}",
+          execResult.exitValue(), execResult.stdout(), execResult.stderr());
+
+      try {
+        Thread.sleep(2 * 1000);
+      } catch (InterruptedException ie) {
+        // ignore
+      }
+    }
+    // check pod is ready
+    checkPodReady(serverName, wdtDomainUid, wdtDomainNamespace);
+
+    // get the restart count of the container in pod after liveness probe restarts
+    int afterRestartCount = assertDoesNotThrow(() ->
+            getContainerRestartCount(wdtDomainNamespace, null, serverName, null),
+        String.format("Failed to get the restart count of the container from pod {0} in namespace {1}",
+            serverName, wdtDomainNamespace));
+    assertTrue(afterRestartCount - beforeRestartCount == 1,
+        String.format("Liveness probe did not start the container in pod {0} in namespace {1}",
+            serverName, wdtDomainNamespace));
+
```

Comment:
After pod restart we have to make sure the deployed applications can continue to be accessed. Please add a check for it.

Reply:
This test is only to verify liveness probe is working.

Reply:
will be adding a check